### ISSUE TITLE
Why define the Variable before start training ?

### ISSUE THREAD

**Issue by QZQTechMonster** [jiasenlu/visDial.pytorch#5]
I have noticed that, in your 'train_*.py' scipt, you always define the Variable before start training.
And in every training step, you use Tensor.resize_() method and Tensor.copy_() method to prepare the Variable.

I am curious about it.
What is the advantage of this trick ? Is this a trick to make the training faster ?
Thank you !
